pandas/compat/numpy/__init__.py

```diff
@@ -62,6 +63,17 @@ def np_array_datetime64_compat(arr, *args, **kwargs):
     return np.array(arr, *args, **kwargs)
 
 
+def np_issubclass_compat(unique_dtype, dtypes_set):
```

Comment:
this wouldn't g in here, better in pandas/core/dtypes/common.py

with a full doc-string & examples

Reply:
this is *almost* is_numeric, but with the twist about np.number

Reply:
I've moved it and added doc string and examples. I'm not aware of any pandas dtypes that have ` _is_numeric = True` and don't inherit from `np.number`.

Should `is_numeric` return True for ExtensionDtypes that have ` _is_numeric = True` and don't inherit from `np.number`?